common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -20,7 +22,7 @@ describe 'Problem', ->
 
     spyOn Logger, 'log'
     spyOn($.fn, 'load').andCallFake (url, callback) ->
-      $(@).html readFixtures('problem_content.html')
+      $(@).html problem_content_default
```

Comment:
I needed to re-render the problem content in my tests, so instead of reading the fixture multiple times, I ended up this way of reading `problem_content.html` fixture once.